File: source/contents/c20_automation/code/p01_playwright/01_playwright.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto("https://www.baidu.com")
    page.fill('#kw', "Playwright")
    page.click('#su')
    page.screenshot(path="baidu.png")
    print(page.title())
    browser.close()

File: source/contents/c20_automation/code/p01_playwright/01_playwright.js
const { chromium } = require('playwright');

(async () => {
    const browser = await chromium.launch();
    const page = await browser.newPage();
    await page.goto('https://www.baidu.com');
    await page.fill('#kw', "Playwright")
    await page.click('#su')
    await page.screenshot({path:"./baidu.png"})
    console.log(await page.title());
    await browser.close();
})();